projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -296,6 +296,7 @@ export class TuiInputDateRangeComponent
 
         if (value.length !== DATE_RANGE_FILLER_LENGTH) {
             this.updateValue(null);
+            this.onOpenChange(true);
```

Comment:
This way it would open if we just type date with a keyboard, that's not what we want to happen.

Reply:
> Maybe we should just drop .stop from cleaner in primitive textfield?
I thought about it, but If open dropdown by click and then click on icon dropdown will close in this case

> This way it would open if we just type date with a keyboard, that's not what we want to happen.
Combobox and multiselect work this way with dropdown, I wanted to do the same. But ok, I got that there is no need here